nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/ESTestClientFactory.java

```diff
@@ -68,29 +64,47 @@ public ESClient create(ElasticSearchEmbeddedNode node, ElasticSearchClientConfig
     protected ESClient createTransportClient(ElasticSearchEmbeddedNode node) {
         ESTransportClientFactory factory = new ESTransportClientFactory();
         ElasticSearchClientConfig config = new ElasticSearchClientConfig();
+        String addressList = System.getProperty(ADDRESS_LIST_PROPERTY);
         addOptions(config);
-        log.info("Using Elasticsearch TransportClient");
-        System.out.println("ElasticSearchClient: TransportClient");
-        return factory.create(node, config);
+        if (addressList == null) {
+            infoMessage("ElasticSearchClient: TransportClient");
+            return factory.create(node, config);
+        } else {
+            infoMessage("ElasticSearchClient: TransportClient on " + addressList);
+            return factory.create(null, config);
+        }
+    }
+
+    protected void infoMessage(String message) {
+        System.out.println(message);
+        log.info(message);
```

Comment:
Don't we want to remove this method and have logger calls instead?

Reply:
There are few `println` done when running tests because we don't want to miss which type of backends are used. I don't want to change this behavior here.